queries/water.jinja2

```diff
@@ -54,6 +99,26 @@ FROM ne_10m_playas
 WHERE {{ bounds|bbox_filter('the_geom') }}
 GROUP BY name
 
+UNION
+
+SELECT
+    NULL AS name,
+    NULL AS area,
+    'playa' AS kind,
+    'naturalearthdata.com' AS source,
+    st_boundary(the_geom) AS __geometry__,
+    gid AS __id__,
+    'yes' AS boundary
+
+FROM
+{% if 4 <= zoom < 7 %}
```

Comment:
Yup, well spotted, thanks! Looks like the _playas_ only kick in at zoom 4, though. Otherwise, fixed in 65f727d.